pkg/logs/launchers/integration/launcher.go

```diff
@@ -136,46 +150,45 @@ func (s *Launcher) makeFileSource(source *sources.LogSource, filepath string) *s
 
 // TODO Change file naming to reflect ID once logs from go interfaces gets merged.
 // createFile creates a file for the logsource
-func (s *Launcher) createFile(source *sources.LogSource) (string, error) {
-	directory, filepath := s.integrationLogFilePath(*source)
+func (s *Launcher) createFile(id string) (string, error) {
+	directory, logFilePath := s.integrationLogFilePath(id)
 
 	err := os.MkdirAll(directory, 0755)
```

Comment:
Note for later, but maybe we could add a check here to see if the directory is already created (or create it right at the start). It seems like all the files will be in the same folder so it might be extraneous to make the directory every time.

Reply:
Directory created at integration start